packages/react-components/src/docs/foundations/Typography.stories.mdx

```diff
@@ -53,25 +62,39 @@ You can modify the output element via `as` prop. For example if you want to rend
     backgroundColor: 'var(--background)',
   }}
 >
-  <Text>Paragraph Medium</Text>
-  <Text bold>Paragraph Medium Bold</Text>
-  <Text underline>Paragraph Medium Underline</Text>
-  <Text strike>Paragraph Medium Strikethrough</Text>
-  <Text size="sm">Paragraph Small</Text>
-  <Text size="sm" bold>
-    Paragraph Small Bold
-  </Text>
-  <Text size="xs">Paragraph XSmall</Text>
-  <Text size="xs" bold>
-    Paragraph XSmall Bold
-  </Text>
-  <Text caps>Caps</Text>
-  <Text caps bold>
-    Bold Caps
-  </Text>
-  <Text as="div" size="sm">
-    Small text as div
-  </Text>
+  <div>
+    <Text>Paragraph MD</Text>
+    <Text bold>Paragraph MD Bold</Text>
+    <Text underline>Paragraph MD Underline</Text>
+    <Text strike>Paragraph MD Strikethrough</Text>
+    <Text caps>Paragraph MD CAPS</Text>
+  </div>
+  <div>
+    <Text size="sm">Paragraph SM</Text>
+    <Text size="sm" bold>
+      Paragraph SM Bold
+    </Text>
+    <Text size="sm" underline>
+      Paragraph SM Underline
+    </Text>
+    <Text size="sm" strike>
+      Paragraph SM Strikethrough
+    </Text>
+    <Text size="sm" caps>Paragraph SM CAPS</Text>
+  </div>
+  <div>
+    <Text size="xs">Paragraph XS</Text>
+    <Text size="xs" bold>
+      Paragraph XS Bold
+    </Text>
+    <Text size="xs" underline>
+      Paragraph XS Underline
+    </Text>
+    <Text size="xs" strike>
+      Paragraph XS Strikethrough
+    </Text>
+    <Text size="xs" caps>Paragraph XS CAPS</Text>
+  </div>
```

Comment:
Please create a Map structure where you can change the size parameter dynamically and not duplicate the code. Something like we have in button stories